### DISCUSSION TITLE
How do I down load Twine?

### DISCUSSION THREAD

**Discussion by Meringlen** [community/community#37988] [New to GitHub]
The Quickstart Guide says, "You can download a local copy of Twine from the homepage".  No kidding!  

HOW?

I clicked "DOWNLLOAD".  Then what?  It didn't download.  I get a new page with lots of stuff.  Don't know what any of this is.  Don't know what to do.

HOW do I download this?

**Comment by LoftyDroid** [comment]
Can you share the homepage here?

**Comment by DSDmark** [comment]
Hi @Meringlen, 

I guess it's not correct to ask topic related to **Twine**. Go there. That is the **Twine discussion** section, ask there:-

[**Twine discussion**](https://twinery.org/forum/discussion/8486/local-save)

**Comment by Meringlen** [comment]
Sorry.  I don't understand the question.

**Reply by DSDmark** [reply]
Are you talking about that question?
> Can you share the homepage here?

**Comment by Meringlen** [comment]
yes

**Comment by Meringlen** [comment]
Also, I seem to have lost all of my work yesterday (for a second time).

The twine window I see does not have an ARCHIVE button anywhere.

**Comment by Meringlen** [comment]
I just realized what you may be asking for.  A Twine user I know went me
this image in answer to my question.

When I go to the Twine website and click DOWNLOAD, I get the page in the
photo.  I had no idea what that was or what I was supposed to do next.  You
should add some instructions for those of us without IT degrees.

Peace,
M

**Comment by Meringlen** [comment]
Why do I not have the panel on the right?

My page does not match the images in the cookbook.
